pkg/config/protocol_config.go

```diff
@@ -122,6 +122,24 @@ func (p *ProtocolConfiguration) Validate() error {
 			return fmt.Errorf("Hardforks configuration section contains unexpected hardfork: %s", name)
 		}
 	}
+	var (
+		prev             uint32
+		shouldBeDisabled bool
+	)
+	for _, cfgHf := range Hardforks {
+		h := p.Hardforks[cfgHf.String()]
+		if h != 0 && shouldBeDisabled {
+			return fmt.Errorf("hardfork %s should be disabled", cfgHf.String())
```

Comment:
It's a really unobvious way to say "missing previous hardfork configuration with cfgHf present`.

Reply:
Fixed.